ocpmodels/models/base.py

```diff
@@ -18,6 +27,49 @@ def __init__(self, num_atoms=None, bond_feat_dim=None, num_targets=None):
     def forward(self, data):
         raise NotImplementedError
 
+    def get_graph_properties(self, data):
```

Comment:
Maybe rename to `generate_graph`?

Reply:
Any other closer suggestions? this would error out for GemNet-OC as that has a method which would override this.

Reply:
You could rename the GemNet-OC one to `generate_graph_dict`..